### ISSUE TITLE
Not a correct displaying for Robots.txt

### ISSUE THREAD

**Issue by Sharkozp** [magento/magento2#13214]
### Preconditions

1. Magento 2.2.1

### Steps to reproduce

1. Add new Robots to Content -> Configuration (Design) -> Edit (Global) -> Search Engine Robots
2. Add this lines:
```
User-agent: *
Disallow: /index.php/
Disallow: /*?
Disallow: /checkout/
Disallow: /app/
Disallow: /lib/
Disallow: /*.php$
Disallow: /pkginfo/
Disallow: /report/
Disallow: /var/
Disallow: /catalog/
Disallow: /customer/
Disallow: /sendfriend/
Disallow: /review/
Disallow: /*SID=
```
3. Or reset to default
4. Check http://{magento_url}/robots.txt

### Expected result

1. It must show the same that added to textfield
```
User-agent: *
Disallow: /index.php/
Disallow: /*?
Disallow: /checkout/
Disallow: /app/
Disallow: /lib/
Disallow: /*.php$
Disallow: /pkginfo/
Disallow: /report/
Disallow: /var/
Disallow: /catalog/
Disallow: /customer/
Disallow: /sendfriend/
Disallow: /review/
Disallow: /*SID=
```

### Actual result

1. It is showing all in one line:
![image](https://user-images.githubusercontent.com/3489232/34987593-f80e6486-fabb-11e7-91d5-e339fba636e3.png)

**Comment by magento-engcom-team** [comment]
@Sharkozp, thank you for your report.
We've acknowledged the issue and added to our backlog.

**Comment by hostep** [comment]
Noticed this too today, I'm pretty sure this is because the content-type is set to `text/html` instead of the expected `text/plain`

**Comment by hostep** [comment]
Just noticed this is already fixed on the `2.2-develop` branch by this PR: https://github.com/magento/magento2/pull/12310

**Comment by johnny-longneck** [comment]
Can confirm this issue on live 2.2.2 site. It only appears when the store view code is added to the url.
For me it is:

www.domain.com/robots.txt <- Correct format, wrong sitemap link
www.domain.com/de/robots.txt <- Wrong format, correct sitemap link

**Comment by okorshenko** [comment]
Hi @Sharkozp the issue has been fixed in 2.2-develop and 2.1-develop branches
Will be available with 2.2.4 release
